docs/network protocol.md

```diff
@@ -127,29 +147,29 @@ InvalidItemsHandling indicates a wrong value type or flag combination was sent.
 Sent to clients when the connection handshake is successfully completed.
 #### Arguments
 | Name              | Type                                     | Notes                                                                                                                                               |
-|-------------------|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| ----------------- | ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
 | team              | int                                      | Your team number. See [NetworkPlayer](#NetworkPlayer) for more info on team number.                                                                 |
 | slot              | int                                      | Your slot number on your team. See [NetworkPlayer](#NetworkPlayer) for more info on the slot number.                                                |
 | players           | list\[[NetworkPlayer](#NetworkPlayer)\]  | List denoting other players in the multiworld, whether connected or not.                                                                            |
 | missing_locations | list\[int\]                              | Contains ids of remaining locations that need to be checked. Useful for trackers, among other things.                                               |
 | checked_locations | list\[int\]                              | Contains ids of all locations that have been checked. Useful for trackers, among other things. Location ids are in the range of ± 2<sup>53</sup>-1. |
-| slot_data         | dict\[str, any\]                         | Contains a json object for slot related data, differs per game. Empty if not required. Not present if slot_data in [Connect](#Connect) is false.    |
-| slot_info         | dict\[int, [NetworkSlot](#NetworkSlot)\] | maps each slot to a [NetworkSlot](#NetworkSlot) information.                                                                                        |
+| slot_data         | Optional\[dict\[str, any\]\]             | Contains a json object for slot related data, differs per game. Empty if not required. Not present if slot_data in [Connect](#Connect) is false.    |
```

Comment:
you sure this is optional? i cant test right now, but i dont be surprised you receive an empty dict

Reply:
https://github.com/ArchipelagoMW/Archipelago/blob/main/MultiServer.py#L1718

Yeah, `slot_data` is only added to the packet if the client requests it to be added. I'm not entirely sure if `Optional[]` is the correct way to document it, as I've heard python still complains if the value is undefined instead of None, but idk what a better way to document it might be.